server/Server.Domain/Commands/ForgetMeCommand.cs

```diff
@@ -0,0 +1,32 @@
+namespace Server.Domain.Commands;
+
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Server.Domain;
+
+public class ForgetMeCommand : ICommand
+{
+    public required string Id { get; init; }
+
+    public async Task Execute(
+        BookRepoContext context,
+        IPublisher publisher,
+        CancellationToken cancellationToken
+    )
+    {
+        if (await context.Customer.FindAsync([ Id ], cancellationToken) is not { } customer)
+        {
+            return;
+        }
+
+        var x = await context
+            .Bookshelves
+            .Where(x => x.CustomerId == Id)
+            .ExecuteDeleteAsync(cancellationToken);
+
+        var y = await context
+            .Customer
+            .Where(x => x.Id == customer.Id)
+            .ExecuteDeleteAsync(cancellationToken);
+    }
+}
```

Comment:
In the future need to get FluentApis so I can do some FluentValidaton on the imports of the models